### ISSUE TITLE
Deployment automation

### ISSUE THREAD

**Issue by Nimbz** [DanWin/hosting#50]
Something like an Ansible playbook to automate deployment

**Comment by linhux** [comment]
Asible are not free for all, so i meen it must free like docker

**Comment by Nimbz** [comment]
@linhux Ansible isn't free for all? Fortunately it's not the only game in town. What should I use instead of Ansible?

**Comment by linhux** [comment]
yes so other user have a easy free to use solution :-) like a livecd or a install script

**Comment by speedmann** [comment]
Ansible is released under GNU General Public License v3.0 therefore it is basically free for everyone with some restrictions.

I have added this issue to my to-do list and might try to create a playbook for this hopefully during this year
